format-all.el

```diff
@@ -620,8 +620,8 @@ Consult the existing formatters for examples of BODY."
       (list "--filename" (buffer-file-name))))))
 
 (define-format-all-formatter rustfmt
-  (:executable "rustfmt")
-  (:install "cargo install rustfmt")
+  (:executable "cargo-fmt")
```

Comment:
I was unsure whether this should be `cargo fmt` or `cargo-fmt`. They are the same binary, `cargo` automatically aliases `cargo-*` to automatically be subcommands (similar to how `git` does things too)`.

Reply:
If they are the same binary, either choice ought to be fine. If there's a convention in the Rust community that one is more supported or official than the other, we can adopt that one, but based on your description it doesn't sound like it makes a difference.